### PR TITLE
Fix theming issues

### LINKED ISSUE #965: setting controlls need some theme work (dark mode)
Especially the text input fields need another border color.

### PR DESCRIPTION
**Pull request by MIntrovrt**
TODO:

- [x] fix gradients on single choice buttons in settings

closes #965

### CODE REVIEW

**Comment by Simon-Laux** [comment]
the emoji menu might need some work, too

**Comment by Simon-Laux** [comment]
+ unblock contacts

**Comment by Simon-Laux** [comment]
> Detect background color and change welcome image to maintain contrast? (only really important for amoled theme)

it an svg, we can change the stroke color somehow in code, and when it is too difficult we can add an alternative image that is inverted

**Comment by MIntrovrt** [comment]
I made a merge conflict, will solve when im done with everything else

File: scss/composer/_emoji-sticker-picker.scss [hunk 1]
```diff
@@ -32,6 +33,10 @@ $emojimart-search-height: 31px + 6px;
   height: calc(#{$esp-header-height} + 1px);
   display: flex;
   border-bottom: 1px solid var(--emojiMartBorder);
+  background: var(--emojiMartBg);
+  border-top-left-radius: var(--emojiMartOutsideRadius);
+  border-top-right-radius: var(--emojiMartOutsideRadius);
+
 }
```
Comment (MIntrovrt) [inline scss/composer/_emoji-sticker-picker.scss]: This Made the a merge conflict. delete neither and just put this in between

Comment (Simon-Laux) [reply scss/composer/_emoji-sticker-picker.scss]: try to rebase your branch to master

File: src/renderer/ThemeBackend.js [hunk 2]
```diff
@@ -160,10 +160,14 @@ export function ThemeDataBuilder (theme) {
     bp3SwitchKnobShadowChecked: '0px 1px 0 0px #c9d4d2d1',
     bp3SpinnerTrack: '#acd4e8',
     bp3SpinnerHead: '#42a5f5',
+    outlineProperties: '1px solid #b9b9b9',
```
Comment (Simon-Laux) [inline src/renderer/ThemeBackend.js]: I would replace `#b9b9b9` by transparent in the light theme / by default

**Pull request merged**